content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -14,27 +14,27 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <div data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
-     class="cmp-accordion"
-     data-cmp-is="accordion"
-     data-cmp-single-expansion="${accordion.singleExpansion}">
-    <div data-sly-test="${accordion.items.size > 0}"
-         data-sly-repeat.item="${accordion.items}"
-         class="cmp-accordion__item"
-         data-cmp-hook-accordion="item"
-         data-cmp-expanded="${item.name in accordion.expandedItems}">
-        <h3 data-sly-element="${accordion.headingElement @ context='elementName'}"
-            class="cmp-accordion__header">
-            <button class="cmp-accordion__button${item.name in accordion.expandedItems ? ' cmp-accordion__button--expanded' : ''}"
-                    data-cmp-hook-accordion="button">
-                <span class="cmp-accordion__title">${item.title}</span>
-                <span class="cmp-accordion__icon"></span>
-            </button>
-        </h3>
-        <div data-sly-resource="${item.name @ decorationTagName='div'}"
-             data-cmp-hook-accordion="panel"
-             class="cmp-accordion__panel${item.name in accordion.expandedItems ? ' cmp-accordion__panel--expanded' : ' cmp-accordion__panel--hidden'}"
-             role="region"></div>
-    </div>
-    <sly data-sly-resource="${resource.path @ resourceType='wcm/foundation/components/parsys/newpar', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent'}"
-         data-sly-test="${(wcmmode.edit || wcmmode.preview) && accordion.items.size < 1}"></sly>
```

Comment:
I'd rather not change the indentation on the core components. In fact, I think you should change your IDE settings to match the indentation already established by Adobe in their code format.